### PR TITLE
Update to java/tomcat-native@1.2.16

### PR DESCRIPTION
**Pull request by mattbishop**
#### Description
Update tomcat-native to newer version

###### Tested on
macOS 10.13.2
Xcode 9.2

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [X] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?
https://trac.macports.org/ticket/55483
- [X] checked your Portfile with `port lint`?
- [X] tried existing tests with `sudo port test`?
- [X] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by mf2k** [comment]
Please add your GitHub handle to the maintainers line.

**Comment by mattbishop** [comment]
@mf2k where is the "maintainers line"?

File: java/tomcat-native/Portfile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 PortSystem 1.0
 
 name                tomcat-native
-version             1.2.14
+version             1.2.16
 categories          java www
 maintainers         thebishops.org:matt
```
Comment (mf2k) [inline java/tomcat-native/Portfile]: This is the maintainers line. So in your case, it should be:
`maintainers	{thebishops.org:matt @mattbishop} openmaintainer`

Comment (mattbishop) [reply java/tomcat-native/Portfile]: IOC

**Comment by mattbishop** [comment]
I've added my GitHub handle to the maintainers list. This should be ready for merging.

**Pull request merged**